test/tests/imageserver/test_image_service_system.py

```diff
@@ -0,0 +1,385 @@
+        mon_url = '/images?name=' + osname + '&version=' + osversion
+        serverip = self._get_serverip()
+        control_port = str(fit_common.fitcfg()["image_service"]["control_port"])
+        response = fit_common.restful("http://" + serverip + ":" + control_port + mon_url, rest_action="delete")
+        if response['status'] in range(200, 205):
+            return response['json']
+        else:
+            logs.error('Incorrect HTTP return code, expected 201-205, got:' + str(response['status']))
+            return "fail"
+
+    def _delete_os_iso(self, isoname):
+        mon_url = '/iso?name=' + isoname
+        serverip = self._get_serverip()
+        control_port = str(fit_common.fitcfg()["image_service"]["control_port"])
+        response = fit_common.restful("http://" + serverip + ":" + control_port + mon_url, rest_action="delete")
+        if response['status'] in range(200, 205):
+            return response['json']
+        else:
+            logs.error('Incorrect HTTP return code, expected 201-205, got:' + str(response['status']))
+            return "fail"
+
+    def _wait_for_task_complete(self, taskid, retries=60):
+        for dummy in range(0, retries):
+            result = fit_common.rackhdapi('/api/2.0/workflows/' + taskid)
+            if result['json']["status"] == 'running' or result['json']["status"] == 'pending':
+                logs.debug("OS Install workflow state: {}".format(result['json']["status"]))
+                fit_common.time.sleep(30)
+            elif result['json']["status"] == 'succeeded':
+                logs.debug("OS Install workflow state: {}".format(result['json']["status"]))
+                return True
+            else:
+                break
+        logs.error("Task failed with the following state: " + result['json']["status"])
+        return False
+
+    def _get_tester_ip(self):
+        serverip = self._get_serverip()
+        monip = fit_common.fitcfg()["rackhd-config"]["apiServerAddress"]
+        cmd = "ping -R -c 1 " + monip + ""
+        (command_output, exitstatus) = pexpect.run(
+            "ssh -q -o StrictHostKeyChecking=no -t " + fit_common.fitcfg()["image_service"]['usr'] + "@" + serverip +
+            " sudo bash -c \\\"" + cmd + "\\\"", withexitstatus=1,
+            events={"assword": fit_common.fitcfg()["image_service"]['pwd'] + "\n"}, timeout=300)
+        uud = command_output.split("\t")
+        myip = uud[1].split("\r\n")[0]
+        logs.debug('My IP address is: ' + myip)
+        return myip
+
+    def _create_esxi_repo(self):
+        logs.debug("create a ESXi repo")
+        for osrepo in fit_common.fitcfg()["image_service"]["os_image"]:
+            if osrepo["osname"] == "ESXi" and osrepo["version"] == "6.0":
+                os_name = osrepo["osname"]
+                os_version = osrepo["version"]
+                http_iso_url = osrepo["url"]
+                self._upload_os_by_network(os_name, os_version, http_iso_url)
+                break
+

```

Comment:
Before break, log.debug（“create ESXi repo successfully”）

At the same level with for statement, logs.debug("Failed to create ESXi repo")